frontend/src/App.tsx

```diff
@@ -21,6 +21,7 @@ function App() {
 
   return (
     <>
+    <div id="overlay-root"></div>
```

Comment:
indentation seems off - is this formatted correctly?

Reply:
wdym

Reply:
```
<>
<div id="overlay-root"></div>
...
 ```
 
should be:

```
<>
  <div id="overlay-root"></div>
  ...
```

Reply:
ohh i thought u meant in the actual website